lib/setupmanagers/qemuhck/qemuhck.rb

```diff
@@ -108,6 +111,33 @@ def initialize_clients_vm
       end
     end
 
+    def write_host_info
+      collect_vms_info
+      File.write("#{@workspace_path}/#{HOST_INFO_LOG_FILE}", compose_host_info)
+    end
+
+    def collect_vms_info
+      @vm_info = []
+      @vm_info << { name: 'studio', vnc_port: @studio_vm.vnc_port }
+      @platform['clients'].each do |(_k, v)|
+        @vm_info << { name: v['name'], vnc_port: @clients_vm[v['name']].vnc_port }
+      end
+    end
+
+    def compose_host_info
+      logs = []
+      logs << "Hostname: #{`hostname`.strip}"
+      logs << "QEMU version: #{`qemu-system-x86_64 --version`.lines.first.strip}"
+      logs << "Linux OS info: #{parse_os_release}"
```

Comment:
I doubt the utility of printing `/etc/os-release`. It is usually obvious what distribution is in use when `uname -a` is printed.